core/esmf-aspect-model-document-generators/src/main/java/org/eclipse/esmf/aspectmodel/generator/openapi/AspectModelOpenApiGenerator.java

```diff
@@ -198,7 +198,7 @@ private void setServers( final ObjectNode objectNode, final String baseUrl, fina
 
    private boolean includePaging( final Aspect aspect, final Optional<PagingOption> pagingType ) {
       return PAGING_GENERATOR.isPagingPossible( aspect )
-            && ( ( pagingType.isPresent() && pagingType.get() != PagingOption.NO_PAGING ) || pagingType.isEmpty() );
+            && ((pagingType.isPresent() && pagingType.get() != PagingOption.NO_PAGING) || pagingType.isEmpty());
```

Comment:
Please revert the code style changes such as this one; see the [conventions](https://github.com/eclipse-esmf/esmf-sdk/blob/main/CONVENTIONS.md#general-conventions) on options for automatic checks.